### ISSUE TITLE
draksetup broken on Ubuntu 18.04

### ISSUE THREAD

**Issue by chivay** [CERT-Polska/drakvuf-sandbox#422]
Installation hangs:
```
root@ubuntu:/home/ccre/Downloads# draksetup install SW_DVD9_Win_Pro_10_20H2.2_32BIT_English_Pro_Ent_EDU_N_MLF_-2_X22-46650.ISO
/opt/venvs/drakrun/lib/python3.8/site-packages/requests/init.py:89: RequestsDependencyWarning: urllib3 (1.26.2) or chardet (3.0.4) doesn't match a supported version!
warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
[2021-01-15 08:59:45,738][INFO] Ensuring that drakrun@* services are stopped...
[2021-01-15 08:59:45,762][INFO] Performing installation...
[2021-01-15 09:00:07,940][INFO] Checking xen-detect...
Running in PV context on Xen V4.14.
[2021-01-15 09:00:07,950][INFO] Testing if xl tool is sane...
```
Reported by @MurbieDev

**Comment by manorit2001** [comment]
Is there anything else that might help figure out what the issue was?

**Comment by MurbieDev** [comment]
This bug was taken from another ticket [here](https://github.com/CERT-Polska/drakvuf-sandbox/issues/418#issuecomment-761092407)

Basically wasn't working on Ubuntu 18.04, so I moved to Debian Buster instead and it worked like a charm. I can try to pull up the more detailed logs if you can specify what you need.

Thanks,
Kyle

**Comment by manorit2001** [comment]
Hi Kyle,

I am just a new person trying to contribute and was just curious about this issue, But looking at the thread which you shared, it doesn't look like I can help much. The issue seems too difficult for me to do.

Thanks for your reply though

**Comment by BonusPlay** [comment]
We're working on a fix/debug tool on #453. Please be patient :)
